src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -28,64 +28,69 @@ internal ModelOperationsCatalog(IHostEnvironment env)
             _env = env;
         }
 
+        private static TransformerChain<ITransformer> CreateChain(ITransformer transformer)
+        {
+            Contracts.AssertValueOrNull(transformer);
+            if (transformer == null)
+                return new TransformerChain<ITransformer>();
+            return new TransformerChain<ITransformer>(transformer);
+        }
+
         /// <summary>
-        /// Save the model to the stream.
+        /// Save a transformer model and the loader used to create its input data to the stream.
         /// </summary>
-        /// <param name="model">The trained model to be saved.</param>
+        /// <param name="model">The trained model to be saved. Note that this can be <see langword="null"/>, as a shorthand
+        /// for an empty data set. Upon loading with <see cref="LoadWithDataLoader(Stream, out IDataLoader{IMultiStreamSource})"/>
+        /// the returned valued will be an empty <see cref="TransformerChain{TLastTransformer}"/>.</param>
```

Comment:
>valued [](start = 25, length = 6)

typo

Reply:
Thanks @abgoswam ... fixed this `valued` here and in three other places.